src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,13 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Describes whether a command mutates the data in the AddressBook.
```

Comment:
It is better to mention it returns true if the `Command` *potentially* mutates the data (header comments specify a contract, so try to be as precise as possible). E.g. Adding a duplicate person will not mutate data but the corresponding `AddCommand` object will still return `true` for this method.

Reply:
When omitting the `@return` tag, try to be more precise with the description (`Returns true if …` instead of `Describes whether…`).